### PR TITLE
Created new contact ui and toastr for the same

### LINKED ISSUE #5: Create a new contact UI

### PR DESCRIPTION
**Pull request by prasidhprabudhan**
Fixes #5.
Screenshot 2021-10-14 at 6 14 57 PM

### CODE REVIEW

File: app/javascript/src/components/Common/MenubarC/NewContactForm.jsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                },
+                {
+                  label: "Value Two",
+                  value: "value2"
+                },
+                {
+                  label: "Value Three",
+                  value: "value3"
+                }
+              ]}
```
Comment (karthiknmenon) [inline app/javascript/src/components/Common/MenubarC/NewContactForm.jsx:65]: Please move this to a `constants.js` file.

File: app/javascript/src/components/Common/MenubarC/NewContactForm.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        description: yup.string().required("Description is required")
+      })}
+    >
+      {() => (
+        <Form>
+          {/* <Input label="Title" name="title" className="mb-6" /> */}
+          <div className="flex flex-row items-center justify-center space-x-8">
+            <Input
+              label="First Name"
+              required={true}
```
Comment (karthiknmenon) [inline app/javascript/src/components/Common/MenubarC/NewContactForm.jsx:26]: With booleans, you don't need to pass `true`. Just passing the `prop` `required` would mark it as true.

File: app/javascript/src/components/Common/MenubarC/NewContactForm.jsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+// import {  Input, Textarea } from "neetoui/formik";
+import * as yup from "yup";
+
+export default function NewContactForm() {
+  return (
+    <Formik
+      initialValues={{
+        title: "",
+        description: ""
+      }}
```
Comment (karthiknmenon) [inline app/javascript/src/components/Common/MenubarC/NewContactForm.jsx:14]: Please move to a `constants` file.

File: app/javascript/src/components/Common/MenubarC/NewContactForm.jsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+      validationSchema={yup.object({
+        title: yup.string().required("Title is required"),
+        description: yup.string().required("Description is required")
+      })}
```
Comment (karthiknmenon) [inline app/javascript/src/components/Common/MenubarC/NewContactForm.jsx:18]: Please move this to a constants file.

File: app/javascript/src/components/Common/MenubarC/NewContactForm.jsx [hunk 5]
```diff
@@ -0,0 +1,90 @@
+import React from "react";
+
+import { Input, Select } from "@bigbinary/neetoui/v2";
+import { Formik, Form } from "formik";
+// import {  Input, Textarea } from "neetoui/formik";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Common/MenubarC/NewContactForm.jsx:5]: Please remove if not required.

File: app/javascript/src/components/Common/MenubarC/NewContactForm.jsx [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+            <Button
+              type="submit"
+              label="Submit"
+              size="large"
+              style="primary"
+              className="ml-2"
+              disabled={isSubmitting}
+              loading={isSubmitting}
+            />
+         </div> */}
```
Comment (karthiknmenon) [inline app/javascript/src/components/Common/MenubarC/NewContactForm.jsx:85]: Please remove if not required.

File: app/javascript/src/components/Common/MenubarC/index.jsx [hunk 7]
```diff
@@ -14,11 +14,13 @@ import { MenuBar, Header } from "@bigbinary/neetoui/v2/layouts";
 
 import burgerMenu from "components/Common/MenubarN/Notes/BurgerMenu.jpg";
 
+import NewContactPane from "./NewContactPane";
+
 const MenubarC = () => {
   // const [searchString, setSearchString] = useState("");
   // const [showMenu, setShowMenu] = useState(false);
```
Comment (karthiknmenon) [inline app/javascript/src/components/Common/MenubarC/index.jsx:21]: Please remove if not required.

File: app/javascript/src/components/Common/MenubarC/DeleteAlert.jsx [hunk 8]
```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+
+import { Modal } from "neetoui";
+
+import notesApi from "apis/notes";
+
+export default function DeleteAlert({ refetch, onClose, selectedNoteIds }) {
```
Comment (karthiknmenon) [inline app/javascript/src/components/Common/MenubarC/DeleteAlert.jsx:7]: `const DeleteAlert = ({ refetch, onClose, selectedNoteIds }) => {}`

File: app/javascript/src/components/Common/MenubarC/NewContactForm.jsx (hunk 2 above)
Comment (prasidhprabudhan) [reply app/javascript/src/components/Common/MenubarC/NewContactForm.jsx:26]: Done.

File: app/javascript/src/components/Common/MenubarC/NewContactForm.jsx (hunk 3 above)
Comment (prasidhprabudhan) [reply app/javascript/src/components/Common/MenubarC/NewContactForm.jsx:14]: Fixed in #25

**Pull request merged**